### PR TITLE
UIU-1955: Export of fees/fines spreadsheet

### PR DESCRIPTION
**Pull request by annamelnyk**
# Description
- created fees/fines EXPORT spreadsheet for single patron
- added EXPORT option to Fees/Fines History 

# Stories
https://issues.folio.org/browse/UIU-1955
https://issues.folio.org/browse/UIU-1958
# Screenshots
**Export button**
image
**Spreadsheet downloading**
image
**Spreadsheet**
image

### CODE REVIEW

File: src/components/data/reports/FeeFineReport.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  constructor({ data, intl: { formatMessage, formatTime } }) {
+    // data model:
+    // data = {
+    //  feeFineActions: [] - GET: 'feefineactions?query=(userId==:{id}&limit=${MAX_RECORDS})',
+    //  accounts: [] - GET: 'accounts?query=(userId==:{id})&limit=${MAX_RECORDS}',
+    //  loans: [] - GET: 'circulation/loans?query=(userId==:{id})&limit=1000',
+    //  servicePoints: [] - props.okapi.currentUser.servicePoints
+    //  patronGroup: 'group' - string,
+    //  user: user - GET: 'users/:{id}',
+    // }
```
Comment (annamelnyk) [inline src/components/data/reports/FeeFineReport.js:47]: Comment will be removed after finishing of [UXPROD-100](https://issues.folio.org/browse/UXPROD-100)

**Review by maximdidenkoepam — APPROVED** [review]

**Review by mkuklis — APPROVED** [review]

File: src/components/data/reports/FeeFineReport.js (hunk 1 above)
Comment (zburke) [reply src/components/data/reports/FeeFineReport.js:51]: Why remove the comment? It's really nice knowing what shape the data will have, and where it came from! Unless there is some specific reason it _shouldn't_ be present, I say leave it!

File: src/components/util/util.js [hunk 2]
```diff
@@ -15,6 +15,26 @@ export function getFullName(user) {
...
 
+export const formatActionDescription = (action) => {
+  return action.typeAction +
+    (action.paymentMethod
+      ? ('-' + action.paymentMethod)
+      : ' '
+    );
+};
+
+export const formatCurrencyAmount = (amount = 0) => parseFloat(amount).toFixed(2);
```
Comment (zburke) [inline src/components/util/util.js:26]: ui-tenant-settings allows tenants to set their [primary currency](https://folio-snapshot.dev.folio.orgc), "for currency symbol to display". Shouldn't this act just like `formatDateAndTime()` below? Pull `formatNumber` out of `intl` the same way you get [`formatTime`](https://github.com/folio-org/ui-users/pull/1611/files#diff-7c8a7254ac9b2f95f8a27158b661f580b7d43f49c6228f04436b4aa5e2acf6c0R42) and the implement this like 
```
export const formatCurrencyAmount = (amount, formatter) => formatter(amount, { style: 'currency' });
```
Apologies if this is out of scope; feel free to ignore it if so. Er, feel free to tell me we need to file a bunch of Jiras to correctly handle currency display if so.

**Review by zburke — APPROVED** [review]
Nice work!

File: src/components/util/util.js (hunk 2 above)
Comment (annamelnyk) [reply src/components/util/util.js:26]: @zburke that's a good idea to create, not a bunch I suppose, but single separate ticker for `ui-users`. There are lots of places of currency amount.

File: src/components/data/reports/FeeFineReport.js (hunk 1 above)
Comment (annamelnyk) [reply src/components/data/reports/FeeFineReport.js:51]: Agree, will leave the comment

File: src/components/util/util.js (hunk 2 above)
Comment (zburke) [reply src/components/util/util.js:26]: Filed https://issues.folio.org/browse/UIU-2026

**Pull request merged**